muxwebserv/main.c

```diff
@@ -239,7 +256,7 @@ void init_navigation_groups() {
     apply_theme_list_drop_down(&theme, ui_droTTYD, options);
     apply_theme_list_drop_down(&theme, ui_droSyncthing, options);
     apply_theme_list_drop_down(&theme, ui_droRSLSync, options);
-    apply_theme_list_drop_down(&theme, ui_droNTP, options);
+    apply_theme_list_drop_down(&theme, ui_droTailscaled, options);
```

Comment:
Whoopsies! I think you meant to add to the drop down list and not remove NTP haha!

Reply:
That's right! Haha